### ISSUE TITLE
Look into handsontable for the listview stuff

### ISSUE THREAD

**Issue by TimGeyssens** [TimGeyssens/UIOMatic#166]
https://github.com/handsontable/handsontable

**Comment by Jogai** [comment]
I think this is going to have issues with the licence.

Alternatives:

- https://github.com/ag-grid/ag-grid (community is MIT licenced)
- https://github.com/fiduswriter/Simple-DataTables
- https://github.com/fooplugins/FooTable

**Comment by TimGeyssens** [comment]
Ok will double check, but since UI-O-Matic is not commercial...

**Comment by TimGeyssens** [comment]
From what I can see: https://github.com/handsontable/handsontable/blob/master/LICENSE.txt it is ok to use the non-commercial-license

**Comment by TimGeyssens** [comment]
Stuff like https://github.com/pyexcel-webwares/django-excel and https://github.com/methnen/m-chart also use it ...

**Comment by Jogai** [comment]
True, but in the end umbraco developers will deliver it along with an umbraco setup to clients, and then you need the saas licence since that says "In other words, you need this license if your product is made available to End Users other than your employees." (the developer licence is "internal use only")

m-chart is using an older version, since 7 its not MIT anymore. This part of the readme was added with v7: https://github.com/handsontable/handsontable#license

**Comment by TimGeyssens** [comment]
ok thanks for the info :)
